backend/services/unified_samurai_agent.py

```
@@ -939,41 +1088,409 @@ async def _handle_processing_error(self, message: str, error: Exception, project
+            except json.JSONDecodeError as e:
+                logger.warning(f"Failed to parse task breakdown JSON: {e}")
+                # Try to extract JSON from the response using regex
+                import re
+                json_pattern = r'\[.*\]'
+                matches = re.findall(json_pattern, response, re.DOTALL)
+                if matches:
+                    try:
+                        parsed_response = json.loads(matches[0])
+                        if isinstance(parsed_response, list):
+                            return parsed_response
+                    except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)